stdlib/public/stubs/Stubs.cpp

```diff
@@ -435,6 +440,287 @@ __mulodi4(di_int a, di_int b, int* overflow)
     }
     return result;
 }
+
+#if defined(_WIN64)
+
+#define CHAR_BIT      8         // number of bits in a char
+
+typedef union
+{
+    tu_int all;
+    struct
+    {
+        du_int low;
+        du_int high;
+    }s;
+} utwords;
```

Comment:
This technically is a GNU extension (union fields may only be access through initialized fields) and breaks aliasing.  At that point, you should assume that the `__mode__` extension is available